overlay/mkcrate.nix

```diff
@@ -167,29 +167,31 @@ let
         echo source = \"registry+''${registry}\" >> Cargo.lock
       fi
       mv Cargo.toml Cargo.original.toml
-      remarshal -if toml -of json Cargo.original.toml \
+
+      yj -t < Cargo.original.toml \
         | jq "{ package: .package
               , lib: .lib
               , bin: .bin
               , test: .test
               , example: .example
               , bench: (if \"$registry\" == \"unknown\" then .bench else null end)
               } + $manifestPatch" \
-        | remarshal -if json -of toml > Cargo.toml
+        | jq '. | walk(if type == "object" then with_entries(select(.value != null)) else . end)' \
```

Comment:
This was a previous attempt at deleting JSON `null` values, but this gave no result. I will drop this.